docs/source/conf.py

```diff
@@ -329,6 +335,54 @@
     ("py:class", "mlflow.models.signature.ModelSignature"),
 ]
 
+
+def _get_reference_map():
+    """
+    Gets a dict that maps an invalid reference to a valid one.
```

Comment:
Could we add some context (e.g. basically just your description in https://github.com/mlflow/mlflow/pull/4242/files#r613161818) to why this is necessary in the docstring, for future readers? Namely that Sphinx computes references using fully-qualified classnames, so references in undocumented modules (even if the referenced object is exposed via a different module from the one it's defined in) are considered invalid by Sphinx (you can use the ActiveRun example)

Reply:
Thanks for the comment! Added some context!